### PR TITLE
Adding information about expanding the query area 

### PR DESCRIPTION
**Pull request by lidiazuin**
As requested in user feedback.

https://trello.com/c/GOM8inXz/

### CODE REVIEW

**Review by AlexicaWright — COMMENTED** [review]
I am not sure if we can substitute all "click" for "select" here, they are not really synonyms. Let's think about the wider implications of this change, we would have to change it everywhere. I agree that in many cases you probably could use "select", but far from all. I try to use to avoid teaching users how to click a button. But in some cases, it's necessary to say that instead of select.

**Comment by lidiazuin** [comment]
I reverted the changes in "to click" so we can do only the requested update and discuss what we are going to do about that.

File: docs/modules/ROOT/pages/operations/result-frames.adoc [hunk 1]
```diff
@@ -45,17 +45,18 @@ Additionally, you can zoom using trackpad zoom gestures or a mouse wheel in comb
...
+* Select a node to get a halo, where you can:
+** Dismiss/hide a node by selecting the image:dismiss-node.png[width=3%] icon.
 ** Expand/collapse child relationships by clicking the icon image:expand-collapse.png[width=3%] icon.
 Alternatively, you can double-click a node.
 ** Unlock the node to re-layout it by clicking the image:unlock-relayout.png[width=3%] icon.
-Alternatively, you can click the node and drag it around.
+Alternatively, you can select the node and drag it around.
 * If you cannot see the whole graph or the results display too close together, you can adjust by moving the visual view and dragging nodes to rearrange them.
-* To move the view to see more parts of the graph, click an empty spot within the graph pane and drag it.
+* To move the view to see more parts of the graph, select an empty spot within the graph pane and drag it.
```
Comment (AlexicaWright) [inline docs/modules/ROOT/pages/operations/result-frames.adoc]:
```suggestion
* To move the view to see more parts of the graph, use an empty spot within the graph pane and drag it.
```

**Review by AlexicaWright — COMMENTED** [review]
Some ideas!

**Comment by lidiazuin** [comment]
Great ideas! I added some manually as they weren't syncing in the PR.

File: docs/modules/ROOT/pages/operations/result-frames.adoc [hunk 2]
```diff
@@ -45,17 +45,18 @@ Additionally, you can zoom using trackpad zoom gestures or a mouse wheel in comb
 (If you are in full-screen view, the modifier key is not needed to zoom.)
 On Mac, use `⌘ + scroll` and on Windows and Linux, use `Ctrl + scroll` to trigger zoom.
 You can also use the _Fit to screen_ button to fit all query results into the view.
+* Expand the Cypher editor area with the image:expand.svg[width=3%] icon next to the play button in the editor.
 * Click a node or a relationship to view its properties.
 The nodes already have sensible captions assigned by the Browser, which auto-selects a property from the property list to use as a caption.
 To change how your graph looks like, see xref:operations/browser-styling.adoc[].
 * Click a node to get a halo, where you can:
```
Comment (AlexicaWright) [inline docs/modules/ROOT/pages/operations/result-frames.adoc:52]:
```suggestion
* Select a node to get a halo, where you can:
```

File: docs/modules/ROOT/pages/operations/result-frames.adoc [hunk 3]
```diff
@@ -45,17 +45,18 @@ Additionally, you can zoom using trackpad zoom gestures or a mouse wheel in comb
 (If you are in full-screen view, the modifier key is not needed to zoom.)
 On Mac, use `⌘ + scroll` and on Windows and Linux, use `Ctrl + scroll` to trigger zoom.
 You can also use the _Fit to screen_ button to fit all query results into the view.
+* Expand the Cypher editor area with the image:expand.svg[width=3%] icon next to the play button in the editor.
 * Click a node or a relationship to view its properties.
 The nodes already have sensible captions assigned by the Browser, which auto-selects a property from the property list to use as a caption.
 To change how your graph looks like, see xref:operations/browser-styling.adoc[].
```
[the remark this answers is not in the document]
Comment (lidiazuin) [reply docs/modules/ROOT/pages/operations/result-frames.adoc]: We could eventually work on a review of the Browser docs, if you think that's worth a project!

File: docs/modules/ROOT/pages/operations/result-frames.adoc [hunk 4]
```diff
@@ -45,17 +45,18 @@ Additionally, you can zoom using trackpad zoom gestures or a mouse wheel in comb
 (If you are in full-screen view, the modifier key is not needed to zoom.)
 On Mac, use `⌘ + scroll` and on Windows and Linux, use `Ctrl + scroll` to trigger zoom.
 You can also use the _Fit to screen_ button to fit all query results into the view.
+* Expand the Cypher editor area with the image:expand.svg[width=3%] icon next to the play button in the editor.
 * Click a node or a relationship to view its properties.
```
Comment (AlexicaWright) [inline docs/modules/ROOT/pages/operations/result-frames.adoc:49]:
```suggestion
* Select a node or a relationship to view its properties.
```

Comment (AlexicaWright) [reply docs/modules/ROOT/pages/operations/result-frames.adoc]: One last click 😆

File: docs/modules/ROOT/pages/operations/result-frames.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply docs/modules/ROOT/pages/operations/result-frames.adoc]: No, I think we'll fix as we go along. Browser will sunset soon anyway.

**Review by AlexicaWright — APPROVED** [review]
This was borderline Pandora's box! Looks good now! Thanks @lidiazuin

**Pull request merged**